UI/UnoMaterialSample/UnoMaterialSample.Skia.Gtk/UnoMaterialSample.Skia.Gtk.csproj

```diff
@@ -14,15 +14,29 @@
     <UpToDateCheckInput Include="..\UnoMaterialSample.Shared\**\*.xaml" />
   </ItemGroup>
   <ItemGroup>
-	<PackageReference Include="Uno.Material" Version="1.3.1" />
-	<PackageReference Include="Uno.Toolkit.UI" Version="1.3.0" />
-    <PackageReference Include="Uno.Toolkit.UI.Material" Version="1.3.0" />
-    <PackageReference Include="Uno.UI" Version="4.1.9" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
-    <PackageReference Include="Uno.UI.Skia.Gtk" Version="4.1.9" />
```

Comment:
You're missing this one in the updated project.

Reply:
Still in the same issue after the update. I'm checking.


![image](https://user-images.githubusercontent.com/5515535/196319921-56b5c03b-0ecd-4eff-98d7-48e232ab756e.png)

Reply:
I'm not seeing this particular package in the changeset, make sure it's there.